### DISCUSSION TITLE
Dualsense on PS2 connects, but don't work

### DISCUSSION THREAD

**Discussion by hardlevel** [darthcloud/BlueRetro#671] [Q&A]
hello, I have a blueretro, that modular one that you can put a cable and connect to the console, I have the firmware updated, when I connect to the ps2, I can pair the dualsense normally, but none of the control buttons work, the led is green, but nothing happens, does anyone have any tips on what I can do?

**Comment by gilgulgamesh** [comment]
Did you find a solution?

**Comment by duna81** [comment]
same here...

**Comment by darthcloud** [comment]
You could try to reload the default config "Default gamepad/keyboard" from the preset page:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#32---mapping-config

If that doesn't work you can try to factory reset the adapter:
https://github.com/darthcloud/BlueRetro/wiki#52---boot-io0-external-adapter (Hold button 10sec)

If that doesn't work then maybe you got a defective adapter or cable.

**Reply by rutdarkronin** [reply]
Green means DualSense is registered as #3 controller, therefore port 1 usage detection pins are low and both 1 & 2 ports considered busy, therefore no input.
